rti/mosaic-rti-api/src/main/java/org/eclipse/mosaic/rti/api/PreemptableFederateAmbassador.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2025 Fraunhofer FOKUS and others. All rights reserved.
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contact: mosaic@fokus.fraunhofer.de
+ */
+
+package org.eclipse.mosaic.rti.api;
+
+public interface PreemptableFederateAmbassador {
+
+    boolean advanceTimePreemptable(long time) throws InternalFederateException;
+
+    boolean processTimeAdvanceGrantPreemptable(long time) throws InternalFederateException;
```

Comment:
Why do we need this method as well? As war as I see, only the `AbstractNetworkAmbassador` which already implements this interface calls this method, no one else. So could it be removed from the interface then?